static/app/views/dashboardsV2/widgetBuilder/buildSteps/yAxisStep/yAxisSelector/index.tsx

```diff
@@ -174,7 +99,11 @@ export function YAxisSelector({
                 displayType,
               })
             }
-            filterAggregateParameters={filterAggregateParameters(fieldValue)}
+            filterAggregateParameters={
+              datasetConfig.filterYAxisAggregateParams
+                ? datasetConfig.filterYAxisAggregateParams(fieldValue, displayType)
+                : undefined
```

Comment:
```suggestion
              datasetConfig.filterYAxisAggregateParams?.(fieldValue, displayType)
```
will return `undefined` if `filterYAxisAggregateParams` isn't defined